pages/profile/permission/views/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import { Link, Snippet, FormLayout } from '@asl/components';
+
+const formatters = {
+  permission: {
+    mapOptions: opt => {
+      return {
+        value: opt,
+        label: <Snippet>{`fields.permission.options.${opt}.label`}</Snippet>,
+        hint: (
+          <Snippet optional>{`fields.permission.options.${opt}.hint`}</Snippet>
+        )
+      };
+    }
+  }
+};
+
+const Page = ({ url }) => {
+  return (
+    <Fragment>
+      <FormLayout formatters={formatters}>
+        <header>
+          <h1>
+            <Snippet>title</Snippet>
+          </h1>
+        </header>
```

Comment:
we have a <Header /> component in @asl/components to inherit shared styles

Reply:
Replaced.